wicker/core/column_files.py

```diff
@@ -90,8 +90,8 @@ class ColumnBytesFileWriter:
 
     def __init__(
         self,
-        storage: S3DataStorage = S3DataStorage(),
-        s3_path_factory: S3PathFactory = S3PathFactory(),
+        storage: AbstractDataStorage = S3DataStorage(),
+        s3_path_factory: WickerPathFactory = S3PathFactory(),  # S3-specific naming kept for backwards compatibility.
```

Comment:
I know this is out of the scope of this PR, but I think the PathFactory should be part of the DataStorage class. Let us see whether we can improve it even a little